test/runtests.jl

```diff
@@ -12,17 +17,19 @@ files = [
     "loggers.jl",
     "io.jl",
     "concurrency.jl",
-    "deprecated.jl",
     "test.jl",
     "ext/json.jl",
 ]
 
 Sys.isunix() ? push!(files, "ext/syslogs.jl") : nothing
 haskey(ENV, "MEMENTO_BENCHMARK") ? files = ["benchmarks.jl"] : nothing
 
-using Memento
+# for loggers.jl
+struct TestError <: Exception
+    msg
+end
 
-cd(dirname(@__FILE__))
+@testset "Memento" begin
```

Comment:
I'm not a fan of having global testsets. Maybe we should switch to using TestSetExtensions.jl?

Reply:
Yes I'll add it in tomorrow